Feature/ld examples

mprorock opened the pull request:
adding some initial examples that we can improve on over time

Sakurann commented on index.html:
```diff
@@ -778,6 +778,22 @@ <h2>Securing <code>application/credential+ld+json</code> with JOSE</h2>
+          class="example"
+          title="A verifiable credential for a University Degree with an external proof"
+        >
+          <p>This verifiable credential is encoded, decode it with <a href="https://jwt.io/">jwt.io</a>.</p>
```
```suggestion
          <p>Below is an example of an encoded verifiable credential.</p>
```

TallTed replied:
```suggestion
          <p>Below is an example of a JWT-encoded verifiable credential, which can be decoded, for example, with <a href="https://jwt.io/">jwt.io</a>.</p>
```

Sakurann commented on index.html:
```diff
@@ -786,6 +802,13 @@ <h2>Securing <code>application/credential+ld+json</code> with COSE</h2>
+          class="example"
+          title="A verifiable credential for a University Degree with an external proof"
+        >
+          <p>This verifiable credential is represented in test vector format used by COSE WG</p>
```
Is there a link to "test vector format used by COSE WG"?

mprorock replied:
```suggestion
          <p>This verifiable credential is represented in test vector format used by COSE as outlined in [[rfc9052]]</p>
```

mprorock commented:
> would like to see above two comments addressed

thanks for the eyes on - I added a cite to the relevant rfc for COSE and incorporated a text improvement from TallTed
